core-server/src/main/java/org/glassfish/jersey/server/model/internal/SseTypeResolver.java

```diff
@@ -0,0 +1,28 @@
+package org.glassfish.jersey.server.model.internal;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+public class SseTypeResolver {
```

Comment:
Please make it final with private constructor

Reply:
fixed